sql_request_abstract/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     "name": "SQL Request Abstract",
-    "version": "14.0.1.0.0",
+    "version": "14.0.1.0.1",
```

Comment:
for your information, that change is not necessary, as  the  oca bot bump the version.